src/controller/datacontroller.js

```diff
@@ -198,10 +198,28 @@ export default class DataController {
 			this.model.selection.removeAllRanges();
 			this.model.selection.clearAttributes();
 
+			// Parse data to model and extract markers from parsed document fragment.
+			const { modelItem, markersData } = this.parse( data );
```

Comment:
I like this idea. Converter still will be returning `Object` with conversion result and markers data, but `parser()` will merge it together into `DocumentFragment` with its own `MarkersCollection` ?